src/components/ApplicationList.tsx

```
@@ -156,19 +156,35 @@ const ApplicationList: React.FC<ApplicationListProps> = ({
 											_hover={{ bg: '#F5F5F5' }}
 											position="relative"
 											pb={paddingBottom}
+											cursor="pointer"
+											role="link"
+											tabIndex={0}
+											aria-label={`View ${app.application_name}`}
+											title={`View ${app.application_name}`}
+											_focusVisible={{
+												outline: '2px solid',
+												outlineColor: '#3c5fdd',
+												outlineOffset: '2px'
```

ESLint

[error] Insert `,`

(prettier/prettier)